src/cli/index.js

```diff
@@ -17,6 +17,6 @@
  * under the License.
  */
 
-require('../apm')();
 require('../setup_node_env');
+require('../apm')();
```

Comment:
What's the reason for moving this call below `setup_node_env`?

We need to be sure this doesn't interfere with our ability to instrument Kibana.

Reply:
@watson I'm trying to get the static-filesystem to work again. I'm experimenting some changes but basically the `src/apm` file was importing a node_module stored in the static-fs, that is only valid after `setup_node_env` ran. Do you think that change anything relevant in the instrumentation ?

Reply:
It depends if any of the modules loaded inside the `setup_node_env` folder is included in the list of modules patched by the APM agent:

https://github.com/elastic/apm-agent-nodejs/blob/cde2dde8c896e3f6f8313fa3db66da88106d6397/lib/instrumentation/index.js#L13-L47

If so, this will break the instrumentation of that particular module.

Reply:
`setup_node_env` should only perform basic setup for our node environment. Looking at the list of the modules I don't think we will include there any of the modules in the list (or others of the same type).

Reply:
If I understand `static-fs` correctly, it will not work trying to load Elastic APM prior to it has done its thing. If you look [towards the bottom](https://github.com/elastic/kibana/blob/38fd963ed3916ccbe565ee22551ce0ff65182a85/src/apm.js#L93) of the `src/apm.js` file you can see that besides `lodash.merge` it also requires `elastic-apm-node`. So removing only `lodash.merge` will not fix the problem. So I think you're right that it has to run after `setup_node_env` in that case. We'll just have to test it properly then

Reply:
I miss that. In that case yes. I'll just remove the changes in `src/apm.js` file